frontend/src/pages/SignupPage/index.js

```diff
@@ -203,14 +218,22 @@ const SignupPage = () => {
         <InterestsForm
           handleChange={handleChange}
           validatesChecked={validatesChecked}
-          extraInterest={state.extraInterest}
-          inputHidden={!state.otherChecked}
           error={state.error}
         />
       ),
       progressBar: <InterestsProgressBar />,
     },
     STEP_3: {
+      buttonText: "Next",
+      component: (
+        <OtherForm
+          extraInterest={state.extraInterest}
+          handleChange={handleChange}
+        />
```

Comment:
Why did you decide here to create an extra component `OtherForm`? Was it just because of `FormLayoutCenter`?

Reply:
It was, handling that right now 😬

Reply:
This is our way to make it go to a new page, as if it is part of a new `STEP`